README.md

```diff
@@ -85,6 +91,10 @@ JavaScript (JS) is the programming language of the web. It allows adding dynamic
 1. [30 day JS challenge](https://javascript30.com/)
 2. [Free code camp](https://www.freecodecamp.org/)
 
+####  Resources
```

Comment:
Please give this a different heading. Something like "utilities" or "libraries".